README.md

```diff
@@ -332,19 +140,27 @@ Attempt to run the language server, and open the log with:
 ```
 Most of the time, the reason for failure is present in the logs.
 
-## Windows
+## Built-in commands
 
-In order for neovim to launch certain executables on Windows, it must append `.cmd` to the command name. A fix is in the works upstream, but until this is mainlined please the following somewhere in your init.vim (lua heredoc) or init.lua:
+* `:LspInfo` shows the status of active and configured language servers.
 
-```lua
-vim.loop.spawn = (function ()
-  local spawn = vim.loop.spawn
-  return function(path, options, on_exit)
-    local full_path = vim.fn.exepath(path)
-    return spawn(full_path, options, on_exit)
-  end
-end)()
-```
+The following support tab-completion for all arguments:
+
+* `:LspStart <config_name>` Start the requested server name. Will only succesfully start if the command detects a root directory matching the current config. Pass `autostart = false` to your `.setup{}` call for a language server if you would like to launch clients solely with this command. Defaults to all servers matching current buffer filetype.
+* `:LspStop <client_id>` Defaults to stopping all buffer clients. 
+* `:LspRestart <client_id>` Defaults to restarting all buffer clients.
+
+## The wiki
+Please see the [wiki](https://github.com/neovim/nvim-lspconfig/wiki) for additional topics, including:
+
+* [Installing language servers automatically](https://github.com/neovim/nvim-lspconfig/wiki/Installing-language-servers-automatically)
+* [Snippets support](https://github.com/neovim/nvim-lspconfig/wiki/Snippets-support)
+* [Project local settings](https://github.com/neovim/nvim-lspconfig/wiki/Project-local-settings)
+* [Recommended plugins for enhanced language server features](https://github.com/neovim/nvim-lspconfig/wiki/Language-specific-plugins)
+
+## Windows
+
+In order for neovim to launch certain executables on Windows, it must append `.cmd` to the command name. To work around this, manually append `.cmd` to the entry `cmd` in a given plugin's setup{} call.  
```

Comment:
I read the `it must append ...` as if neovim was doing it. Rephrasing it so that it is clearer it is the user's responisbility`You may need to append `.cmd` to the cmd entry on windows otherwise neovim wont be able to launch the executable.` . Maybe this is a check that could be done in checkhealth/LspInfo (hoping a windows user reads this).

Reply:
Done in r3 PR, but we should fix this somehow... I had a workaround but it breaks packer, there's an upstream issue in core about it.